### PR TITLE
Adding new docs for event banner alert and switch

### PR DESCRIPTION
**Pull request by olliecurtis**
# **Banner Alert**
Banner Alert

# **Switch**
Switch

### CODE REVIEW

**Comment by olliecurtis** [comment]
@shaundon resolved 😄

**Comment by olliecurtis** [comment]
@shaundon - Addressed what do you think?

File: docs/src/pages/WebSwitchPage/WebSwitchPage.js [hunk 1]
```diff
@@ -43,6 +43,22 @@ const components = [
...
     ],
   },
+  {
+    id: 'event',
+    title: 'Event',
+    blurb: [
+      <Paragraph>
+        Reserved for event-based UI that has neither a positive or negative
+        connotation, for example fare flexibility brought on by a global event,
+        or a weather event.
```
Comment (shaundon) [inline docs/src/pages/WebSwitchPage/WebSwitchPage.js:53]: Can you make it use the same text as the banner alert? It's my bad for not updating the card, not your fault:

> Reserved for event-based messaging that has neither a positive or negative connotation. Avoid using this in other contexts.

Comment (olliecurtis) [reply docs/src/pages/WebSwitchPage/WebSwitchPage.js]: Done!

**Comment by olliecurtis** [comment]
Build passed here but looks like there is an issue between travis and git for reporting the pass

https://travis-ci.org/github/Skyscanner/backpack-docs/builds/691358365

**Pull request merged**